target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -132,13 +132,12 @@ EOF
 }
 
 # Adjust Postfix's configuration files. We only need to append Rspamd at the end of
-# `smtpd_milters` in `/etc/postfix/main.cf`.
+# `smtpd_milters` in `${POSTFIX_MAIN_CF}`.
```

Comment:
Was that change intended?

```suggestion
# `smtpd_milters` in `/etc/postfix/main.cf`.
```

Reply:
Actually, yes. In `postfix.sh`, I introduced `readonly POSTFIX_MAIN_CF=/etc/postfix/main.cf` and in case this ever changes (most unlikely, I know), I wanted to use the value that will always hold. If you think this is not a good idea, we can revert, I do not insist on this change.